### PR TITLE
Renaming "Mis nominaciones de calidad" to "Mis reportes de problemas

### PR DESCRIPTION
**Pull request by Kaweees**
# Descripción

Fixed the readings source to be consistent.

Changed:
"Mis nominaciones de calidad" to "Mis reportes de problemas
modified the string: navMyQualityNomination 
modified the string: nominationsRangeHeader 
Fixes: #4009

### CODE REVIEW

**Comment by lhchavez** [comment]
all right, the only thing missing is to run the command that fixes the errors:

https://github.com/omegaup/omegaup/pull/4402/checks?check_run_id=885975424#step:6:27

File: frontend/templates/en.lang [hunk 1]
```diff
@@ -564,7 +564,7 @@ navViewBadges = "View badges"
 navViewProfile = "My profile"
 newPublicTagsNotAllowed = "It is not allowed to add new public tags"
 noCoders = "No coders."
-nominationsRangeHeader = "Quality nominations (from %(lowCount) to %(highCount))"
+nominationsRangeHeader = "Problem reports"
```
Comment (heduenas) [inline frontend/templates/en.lang:567]: Can you please keep the " (from %(lowCount) to %(highCount))" part at the end of this string?
Same for other languages.

Comment (Kaweees) [reply frontend/templates/en.lang:567]: > all right, the only thing missing is to run the command that fixes the errors:
> 
> https://github.com/omegaup/omegaup/pull/4402/checks?check_run_id=885975424#step:6:27
In which directory do I run this?

Comment (lhchavez) [reply frontend/templates/en.lang:567]: normally all commands are run from the directory where you cloned the omegaup repository on your machine.

Comment (heduenas) [reply frontend/templates/en.lang:567]: try running from `/opt/omegaup/`

**Comment by lhchavez** [comment]
Merged as #4479.

**Pull request closed without merging**